doc/release-notes-1.8.rst

```diff
@@ -0,0 +1,428 @@
+* :jira:`ZEP-2029` - xtensa: irq_offload() doesn't work on XRC_D2PM
+* :jira:`ZEP-2033` - Channel Selection Algorithm #2
+* :jira:`ZEP-2034` - High Duty Cycle Non-Connectable Advertising
+* :jira:`ZEP-2037` - Malformed echo response
+* :jira:`ZEP-2048` - Change UART "baud-rate" property to "current-speed"
+* :jira:`ZEP-2051` - Move away from C99 types to zephyr defined types
+* :jira:`ZEP-2052` - arm: unhandled exceptions in thread take down entire system
+* :jira:`ZEP-2055` - Add README.rst in the root of the project for github
+* :jira:`ZEP-2057` - crash in tests/net/rpl on qemu_x86 causing intermittent sanitycheck failure
+* :jira:`ZEP-2061` - samples/net/dns_resolve networking setup/README is confusing
+* :jira:`ZEP-2064` - RFC: Making net_shell command handlers reusable
+* :jira:`ZEP-2065` - struct dns_addrinfo has unused fields
+* :jira:`ZEP-2066` - nitpick: SOCK_STREAM/SOCK_DGRAM values swapped compared to most OSes
+* :jira:`ZEP-2069` - samples: net: dhcpv4_client: runs failed on frdm k64f board
+* :jira:`ZEP-2070` - net pkt doesn't full unref after send a data form bluetooth's ipsp
+* :jira:`ZEP-2076` - samples: net: coaps_server: build failed
+* :jira:`ZEP-2077` - Fix IID when using CONFIG_NET_L2_BLUETOOTH_ZEP1656
+* :jira:`ZEP-2080` - No reply from RPL node after 20-30 minutes.
+* :jira:`ZEP-2092` - [NRF][BT] Makefile:946: recipe for target 'include/generated/generated_dts_board.h' failed
+* :jira:`ZEP-2114` - tests/kernel/fatal : Fail for QC1000/arc
+* :jira:`ZEP-2125` - Compilation error when UART1 port is enabled via menuconfig
+* :jira:`ZEP-2132` - Build samples/bluetooth/hci_uart fail
+* :jira:`ZEP-2138` - Static code scan (coverity) issues seen
+* :jira:`ZEP-2143` - Compilation Error on Windows 10 with MSYS2
+* :jira:`ZEP-2152` - Xtensa crashes on startup for cores with coprocessors
+* :jira:`ZEP-2178` - Static code scan (coverity) issues seen
+* :jira:`ZEP-2181` - HTTP 400 Bad request forever
+* :jira:`ZEP-2200` - BLE central error "ATT Timeout" when subscribing to NOTIFY attr  
+* :jira:`ZEP-2208` - Coverity static scan issues seen
+* :jira:`ZEP-2211` - Makefile.toolchain.gccarmemb sets DTC to /usr/bin/dtc which is not available on macOS
+* :jira:`ZEP-2219` - Infinite loop caused by NA with unkown ICMPv6 option type (0xff)
+* :jira:`ZEP-2221` - irq_offload() unimplemented for Cortex M0 boards
+* :jira:`ZEP-2222` - [nrf 51] Zephyr fails to compile if TICKLESS_KERNEL is enabled for nrf51
+* :jira:`ZEP-2223` - samples: net: http_client: Cannot send OPTIONS/HEAD request on FRDM K64F
+* :jira:`ZEP-2224` - kernel/mem_pool: test_mpool_alloc_free_isr hits memory crash
+* :jira:`ZEP-2235` - Coverity static scan issues seen
+* :jira:`ZEP-2240` - kernel/test_build/test_newlib / fails for stm32373c_eval with ARM cross-compiler
+* :jira:`ZEP-2257` - test_context fails on bbc_microbit
+ :jira:`ZEP-472` -	Ethernet packets are getting missed if sent in quick succession.
```

Comment:
Not a valid list character -- use an asterisk (*) here and below

Also we'll need to generate this list again right before we do the release to catch any last-minute Jira issues (and to get a sorted list) -- the ones you've added aren't in sorted order with the others.